core/src/mindustry/entities/bullet/BulletType.java

```diff
@@ -192,7 +195,10 @@ public void hit(Bullet b, float x, float y){
 
             if(status == StatusEffects.burning) {
                 indexer.eachBlock(null, x, y, splashDamageRadius, other -> other.team != b.team, other -> {
-                    Damage.createIncend(x, y, damage / 10f, (int)damage / 10);
+                    Tile tile = world.tileWorld(other.x, other.y);
```

Comment:
this can't be null, because `other` isn't null: use `other.tile`

Reply:
changed it to `Fires.create(other.tile);`